doc/decisions/record-storage.md

```diff
@@ -0,0 +1,51 @@
+# Record Storage
+
+## Problem
+
+We need a way to store session recordings.
+Sometimes it is required to run the `kdb` application with `sudo`, especially when changing system-wide configuration.
+Session recordings should be on a per-user basis. Storing them in the `user:/` namespace is not compatible with `sudo`.
+
+## Constraints
+
+We need to support cases where kdb is run as `sudo`
+
+## Assumptions
+
+## Considered Alternatives
+
+## Decision
+
+**Solution 1:** Store session in `system:/` namespace and mount each session as a file in the `/tmp/` directory.
+
+Pros:
+ - Storage layer already implemented, reuse of existing functionality
+ - Can query recordings with existing tooling
+
+Cons: 
+ - Still needs workaround to start session (aka mount a session in `system:/`)
```

Comment:
Do you mean here that `kdb record start` would need to be run as root? If it needs to mount something by design, I do not see a "workaround".